packages/search-types/src/tagging.model.ts

```diff
@@ -1,12 +1,40 @@
 /**
- * The tagging model is a URL with a record of parameters. This tagging is used to track user
- * actions (query, click, show...) by making a request with these info.
+ * A taggable entity.
+ *
+ * @public
+ */
+export interface Taggable {
+  /** Tagging object containing the different taggable events. */
+  tagging: Tagging;
+}
+
+/**
+ * The tagging user actions.
  *
  * @public
  */
 export interface Tagging {
+  /** {@link TaggingInfo | add to cart tagging}. */
+  add2cart?: TaggingInfo;
+  /** {@link TaggingInfo | checkout tagging}. */
+  checkout?: TaggingInfo;
+  /** {@link TaggingInfo | click tagging}. */
+  click?: TaggingInfo;
+  /** {@link TaggingInfo | query tagging}. */
+  query?: TaggingInfo;
```

Comment:
Would it be worth to also add the `wishlisht` by default?

Reply:
Is ok for me.